tokio/src/runtime/blocking/pool.rs

```diff
@@ -91,6 +94,10 @@ where
 impl BlockingPool {
     pub(crate) fn new(builder: &Builder, thread_cap: usize) -> BlockingPool {
         let (shutdown_tx, shutdown_rx) = shutdown::channel();
+        #[cfg(feature = "blocking")]
+        let keep_alive = builder.keep_alive.unwrap_or(KEEP_ALIVE);
+        #[cfg(not(feature = "blocking"))]
+        let keep_alive = KEEP_ALIVE;
```

Comment:
Rather than having to do this, we could just make the `Builder` `keep_alive` field always exist, regardless of the `blocking` feature. It's really just the public API method to _set_ that value that needs to be feature flagged, IMO. 

I don't care a whole lot, but I think not feature flagging the `Builder` field would make this code a bit simpler.

Reply:
Do we want to allow dead code there?